packages/devtools_app/lib/src/framework/status_line.dart

```diff
@@ -98,29 +98,27 @@ class StatusLine extends StatelessWidget {
     MediaSize screenWidth,
   ) {
     final String? docPageId = currentScreen.docPageId;
-    if (docPageId != null) {
-      return RichText(
-        text: LinkTextSpan(
-          link: Link(
-            display: screenWidth <= MediaSize.xs
-                ? docPageId
-                : 'flutter.dev/devtools/$docPageId',
-            url: 'https://flutter.dev/devtools/$docPageId',
-            gaScreenName: currentScreen.screenId,
-            gaSelectedItemDescription: analytics_constants.documentationLink,
-          ),
-          context: context,
-        ),
-      );
-    } else {
-      // Use a placeholder for pages with no explicit documentation.
```

Comment:
was the comment automatically removed? we should probably leave this even with the ternary operator